data.sql

```diff
@@ -104,4 +104,21 @@ VALUES
     (9, 2, '2020-08-03'),
     (10, 3, '2020-05-24'),
     (10, 1, '2021-01-11');
-    
\ No newline at end of file
+
+INSERT INTO
+    visits (animal_id, vet_id, date_of_visit)
+    SELECT * FROM (SELECT id FROM animals) animal_ids,
+    (SELECT id FROM vets) vets_ids,
+    generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+insert into
+    owners (full_name, email)
+    select 'Owner ' ||
+    generate_series(1,2500000), 'owner_' ||
+    generate_series(1,2500000) || '@mail.com';
+
+INSERT INTO
+    visits (animal_id, vet_id, date_of_visit)
+    SELECT * FROM (SELECT id FROM animals) animal_ids,
+    (SELECT id FROM vets) vets_ids,
+    generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
```

Comment:
- [ ] Great work inserting data the way it was recommended. However, I noticed in your screenshots that the execution time before improving performance is significantly lower than **_1 second (1000ms)_**. I would advise you to run this query two or three times, make the `explain analyze` statement and take screenshots of the execution time that are more than _**1 second (1000ms)**_. 👍 😎 

#### You can check points _**3 and 4**_ of this [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) 💻

Reply:
@Gambit142, changes done